src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator has started work.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");
```

Comment:
Similarly, the message should be more informative to reflect that it is an Excavator that has stopped working.